doc/cascadia/profiles.schema.json

```diff
@@ -1969,6 +1969,16 @@
           ],
           "type": "string"
         },
+        "newTabPosition": {
+          "default": "atTheEnd",
+          "description": "Specifies position of the new tab. Possible values are \"atTheEnd\" and \"afterCurrentTab\".",
+          "enum": [
+            "atTheEnd",
+            "afterCurrentTab"
+          ],
+          "type": "string"
+        }
+      },
```

Comment:
This appears to be one-too-many closing braces. VSCode is complaining:
```
Unable to parse content from 'https://raw.githubusercontent.com/microsoft/terminal/2f58711ee6b27064d11dec84010353e144f170f0/doc/cascadia/profiles.schema.json': Parse error at offset 81576.
```
but does not complain about the previous commit.

Reply:
I think this happed with the merge from main